components/InfectionMedicalcareprovisionStatus.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <div class="InfectionMedicalcareprovisionStatus">
+    <div class="InfectionMedicalcareprovisionStatus-heading">
+      <h3 class="InfectionMedicalcareprovisionStatus-title">
+        {{ $t('感染状況・医療提供体制（サマリ）') }}
+        {{ date }}時点
+      </h3>
+    </div>
+    <div class="InfectionMedicalcareprovisionStatus-Box">
+      <div class="InfectionMedicalcareprovisionStatus-Headline">
+        <app-link
+          to="https://www.fukushihoken.metro.tokyo.lg.jp/hodo/saishin/hassei.html"
+        >
+          {{ $t('感染状況') }}
+        </app-link>
+      </div>
+      <div class="InfectionMedicalcareprovisionStatus-description">
+        {{ $t('新規陽性者')
+        }}<span>{{ statuses.data['新規陽性者'].toLocaleString() }}人</span> /
+        {{ $t('検査数')
+        }}<span>{{ statuses.data['検査数'].toLocaleString() }}件</span>（{{
+          statisticDate
+        }}{{ $t('参考値') }}）、 {{ $t('うち65歳以上の高齢者数')
+        }}<span
+          >{{
+            statuses.data['うち65歳以上の高齢者数'].toLocaleString()
+          }}人</span
+        >、 {{ $t('死亡者数')
+        }}<span>{{ statuses.data['死亡者数'].toLocaleString() }}人</span>、
+        {{ $t('都外からの持込検体による陽性数')
+        }}<span>{{
+          statuses.data['都外からの持込検体による陽性数'].toLocaleString()
+        }}</span>
+      </div>
+    </div>
+    <div class="InfectionMedicalcareprovisionStatus-Box">
+      <div class="InfectionMedicalcareprovisionStatus-Headline">
+        <app-link
+          to="https://stopcovid19.metro.tokyo.lg.jp/cards/details-of-confirmed-cases/"
```

Comment:
内部リンクなので、以下のようにできそうです。

```suggestion
          to="/cards/details-of-confirmed-cases/"
```